.github/workflows/release.yml

```diff
@@ -0,0 +1,95 @@
+name: 'release'
+
+on:
+  push:
+    branches:
+    - 'main'
+
+jobs:
+  # build compiles the code and creates a pull request of the compiled result if
+  # there is a diff.
+  build:
+    runs-on: 'ubuntu-latest'
+    steps:
+    - uses: 'actions/checkout@v2'
+
+    - uses: 'actions/setup-node@v2'
+      with:
+        node-version: '12.x'
+
+    - name: 'npm build'
+      run: 'npm ci && npm run build'
+
```

Comment:
Any particular reason for dropping the PR body generation - it lets us see what changes are in a given PR 
https://github.com/google-github-actions/get-gke-credentials/pull/162

Reply:
Two reasons:

1. It's a lot of really fragile git commands. Since this is switching the author to the GitHub Actions bot, it also wouldn't work for awhile.
2. I didn't find the change list particularly useful, especially since I tend to always `npm run build` in the PR. That often meant the list was actually wrong anyway.

Reply:
Makes sense